app/assets/javascripts/initializers/initializeReadingListIcons.js

```diff
@@ -116,6 +118,29 @@ function properButtonFromEvent(event) {
   return properElement;
 }
 
+function addHoverEffectToReadingListButton(button) {
+  if (button.classList.contains('selected')) {
+    button.addEventListener('mouseenter', readingListButtonMouseEnter);
```

Comment:
Any reason `'mouseover` and `mouseout` aren't being used? Also, if you use them, you can  attach them just to the buttons' container and since they bubble, you can act accordingly. For an example of what I mean, see https://dev.to/nickytonline/comment/29df.

Reply:
Thanks @nickytonline! I never know which of the two sets I'm supposed to use. I couldn't find compatibility data for `mouseover` on MDN but I did for `mouseenter` so I went that way. I tried using bubbling but it didn't work with `mouseenter`, that's why :-) I'll try to rewrite this using `mouseover` and bubbling, thanks!

Reply:
@nickytonline done, you can check the following commit to see the change: https://github.com/thepracticaldev/dev.to/pull/2759/commits/83671c16909e5b66acf76c67ad7eb6d86a3cd1c5